video/out/filter_kernels.c

```diff
@@ -303,6 +383,10 @@ const struct filter_kernel mp_filter_kernels[] = {
     {"sinc3",          3,   sinc},
     {"sinc4",          4,   sinc},
     {"sinc",           -1,  sinc},
+    {"ewa_lanczos16",  2,   ewa_lanczos,    .polar = true},
```

Comment:
We could easily fix the other entries in a backward compatible way by adding duplicate entries.